### DISCUSSION TITLE
404 page on tags

### DISCUSSION THREAD

**Discussion by wcgordon1** [satnaing/astro-paper#147] [General]
Hi all!

I deployed to Netlify and Vercel. Still getting a 404 page when I click on tags (from the navbar & blog post). 

Tags page works fine when running on local server. Can't find an error in the code. Looked at Astro Docs.

Can't figure this out! Please help if you can. Thanks so much!

**Comment by MrGKanev** [comment]
It may be something you have changed in your branch. I have a nearly identical to the "stock" template and it works for me. - hosted using Netlify

**Reply by wcgordon1** [reply]
Thanks. Must have been. I forked this repo and it works. before I created a new project from the command line. Appreciate it!
